skimage/morphology/tests/test_binary.py

```diff
@@ -45,6 +45,14 @@ def test_binary_opening():
     grey_res = img_as_bool(grey.opening(bw_img, strel))
     testing.assert_array_equal(binary_res, grey_res)
 
+def test_binary_perimeter():
+    bw = np.zeros([10,10])
+    bw[3:7,3:7] = 1
+    perim = np.copy(bw)
+    perim[4:6,4:6] = 0
+    strel = selem.square(3)
+    binary_res = binary.binary_perimeter(bw, strel)
```

Comment:
you could also write the test with the default structuring element, the result would be the same.